### PR TITLE
Bookstore setup

### PR DESCRIPTION
**Pull request by MarkoKermi**
- [ ] Setup a new React application.

- [ ] Created a display list of books.

- [ ] Create a `Navbar` component that includes <Link>s to the two routes you created.

- [ ] Create a `Book` component that displays a single book (props are title and author.

- [ ] Create a `Form` component.

### CODE REVIEW

**Comment by Kamba56** [comment]
Hi @MarkoKermi

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that **does not have a professional readme file. Kindly add a readme file customized to your project.**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const Categories = () => <p />;
```
Comment (Ismailco) [inline src/components/Categories.js]: - [ ] The `Categories` component should have a `<Button>` with the text "Check status", kindly make sure to implement this so users can check the status (For now this is just a button with no functionalities)

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @MarkoKermi, 

Good job so far 👍🏼 
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ Used the correct linter configurations
✔️ Used Gitflow

## Required Changes ♻️
- [ ] Kindly make sure to change the name of the license in your repo from `MIT.md` to `LICENSE.md` to fix the broken link in your README file

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
✔️ Check status button is present
✔️ App runs as expected
✔️ No linter errors
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] Like the previous reviewer mentioned, kindly fix your MIT license as this would make your repo more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @MarkoKermi, 

To highlight 👍🏽 
✔️ License is fixed
✔️ README is professional

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**